### PR TITLE
configure mysql and phpmyadmin in dev and prod

### PR DESCRIPTION
**Pull request by ramonsaboya**
This pr will configure both dev and prod infrastructures to include mysql and phpmyadmin services, as we will be needing those in the near future.

A few other changes included in this pr:

- change rabbitmq image to "alpine" instead of "management-alpine", as we do not use the management interface
- include https redirect for every subdomain on nginx rules
- remove auth directives from some of the nginx rules, as this is handled by kubernetes ingress
- modified start script to also apply volumes file and remove linkerd from ingress, as it's not handled by linkerd

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -28,6 +14,21 @@ services:
...
     networks:
     - net
+  logger:
+    container_name: logger
+    build:
+      context: logger/.
+      dockerfile: Dockerfile.dev
+    image: dev-logger
+    restart: always
+    environment:
```
Comment (Arthurlpgc) [inline docker-compose.yml:24]: This changes should be in separate PRs..

Comment (Arthurlpgc) [reply docker-compose.yml:24]: This made the changes in this file unreadable, can't see what actually changed

File: nginx/sqlite.conf [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
     listen 80;
     server_name sqlite.vitaes.io;
+
+    if ($http_x_forwarded_proto = "http") {
+        return 301 https://$server_name$request_uri;
+    }
+
     location ~ .* {
-      auth_basic           "Vitaes' Area";
-      auth_basic_user_file /etc/apache2/.htpasswd;
```
Comment (Arthurlpgc) [inline nginx/sqlite.conf:6]: This would make everything readable by everyone,which is not something we want

File: docker-compose.yml (hunk 1 above)
Comment (ramonsaboya) [reply docker-compose.yml:24]: Fair enough, I was being lazy xD
